src/models/hooks/activityReportGoalFieldResponse.js

```diff
@@ -32,7 +32,7 @@ const recalculateOnAR = async (
   instance,
 ) => sequelize.models.GoalFieldResponse.update(
   {
-    onAR: Sequelize.literal(`(
+    onAR: Sequelize.literal(`COALESCE ((
```

Comment:
how is it coming back with null, this works. but it is likely pointing to another issue somewhere else.

Reply:
You are correct - the bug was in the other file, on the front end. The open goal was not being included in the data being passed up, which meant that the backend would attempt to delete that goal as part of the cleanup of "saveGoalsForReport." That's why this code was being triggered, as the goal was being deleted so would it's responses. 

I didn't think it would hurt to leave the COALESCE in but we don't need it to fix the bug.